from curl_cffi.requests.exceptions import HTTPError


class HTTPError(HTTPError):
    pass
